doc/manual/02-installation.md

```diff
@@ -198,3 +198,56 @@ where `$HOME` is your home directory.
 ### Build
 
 These packages will be detected during `cmake` and built as part of the main build.
+
+## Extension installation
+
+When you configure the `extra` parameter in the server section of `pgmoneta.conf`, it requires the server side to have the [pgmoneta_ext][pgmoneta_ext] extension installed to make it work.
+
+The following instructions can help you easily install `pgmoneta_ext`. If you encounter any problems, please refer to the more detailed instructions in the [DEVELOPERS][ext_developers] documentation.
+
+### Install pgmoneta_ext
+
+After you have successfully installed `pgmoneta`, the following commands will help you install `pgmoneta_ext`:
+
+```sh
+dnf install -y pgmoneta_ext
+```
+
```

Comment:
You need the postgrersql.conf `shared_preload_libraries = 'pgmoneta_ext' part as well.

And, describe that it requires a restart of PostgreSQL

Reply:
So, it should be something like this:

```sh
dnf install -y pgmoneta_ext
```

You need to add the `pgmoneta_ext` library for PostgreSQL in `postgrersql.conf` as well:

```ini
shared_preload_libraries = 'pgmoneta_ext'
```

And remember to restart PostgreSQL to make it work.

Reply:
yeah

Reply:
ok, thanks